src/components/ClustersList/index.jsx

```diff
@@ -16,34 +17,58 @@ class ClustersList extends Component {
   }
 
   render() {
-    const { clusters, isRetrieved } = this.props;
+    const { clusters, isRetrieved, isRetrieving } = this.props;
 
     return (
       <div className="ClusterList">
-        {isRetrieved ? (clusters.clusters.map((cluster) => (
-          <Link to={{ pathname: `/clusters/${cluster.id}/resources` }} key={cluster.id}>
-            <div key={cluster.id} className="ClusterCardItem">
-              <ClusterCard name={cluster.name} description={cluster.host} icon={crane} />
+        {
+          isRetrieving ? (
+            <tr className="TableLoading">
```

Comment:
@Mubangizi kindly help me understand why we have table row tags minus the actual table in this file?